.circleci/config.yml

```diff
@@ -145,7 +145,79 @@ jobs:
           path: ~/go/src/github.com/mattermost/mattermost-server/test-results/
       - store_artifacts:
           path: ~/go/src/github.com/mattermost/mattermost-server/test-results/
+      - run:
```

Comment:
maybe instead you make this tests check inside the test step can you create another stage to run this? and make the correct chain, maybe this can run in parallel with the tests

Reply:
thanks!!

Reply:
Done